app/controllers/admin/companies_controller.rb

```diff
@@ -7,4 +7,11 @@ def destroy
     Company.find(params[:id]).destroy
     redirect_to admin_dashboard_index_path
   end
+
+  def update
+    company = Company.find(params[:id])
+    company.update(status: params[:company][:status])
+    company.save
```

Comment:
I think that is a pretty safe bet that we don't need a conditional. I just tried to brainstorm any reason why it would fail and I can't think of any? If we make any changes to allow for an update on anything else besides the status, then we can add in a conditional as fit.